package exception;

public class ReturnInExceptionExample {

	public static void main(String[] args) {
		
	}
	
	public static int test()
	{
	
		
	}
}
